src/assets/Keycloak/oCIS-with-Keycloak.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Case Study About Integrating oCIS With Keycloak
+authorName: Amrita Shrestha
+authorAvatar: https://avatars.githubusercontent.com/u/54478846?s=100&v=4
+authorLink: https://github.com/amrita-shrestha
+createdAt: May 14, 2024
+tags: ownCloud Infinite Scale, oCIS, oCIS-with-Keycloak, Keycloak
+banner: https://blog.jankaritech.com/src/assets/Keycloak/images/oCIS-Keycloak/oCIS-with-Keycloak.png
+
+[In the previous blog](/src/assets/Keycloak/Keycloak.md), we explored the general functionality of Keycloak. In this current post, I will show how to setup of the [oCIS](https://owncloud.dev/ocis/)(file-sync and share platform) with Keycloak.
+
```

Comment:
Sorry, but my blog is about an alternative manual way to set up oCIS with Keycloak. My blog will be helpful for those who have some basic knowledge about keycloak and want to learn Keycloak with practical examples. There are other similar blogs that integrate Keycloak with Nextcloud, GitHub etc. The only difference is that, I use oCIS for integration.

Reply:
I think this suites `Integrating oCIS With Keycloak` best for now lets not invest enormous amount of time on such thing

Reply:
Choosing a suitable title and a proper introduction are some very important aspect of a blog, because the readers usually determine by these two things if it's worth their time to read the rest of the blog :)

Reply:
This blog is about integrating oCIS and Keycloak which seems clear by title. I like to keep it like this for now.